src/centralized_telescope/setup.rs

```diff
@@ -0,0 +1,18 @@
+//! ALBA's Setup structure comprising, among others, the number of elements and internal parameters
+
+/// Setup output parameters
+#[derive(Debug, Clone, Copy)]
+pub struct Setup {
+    /// Approximate size of prover set to lower bound
```

Comment:
I'd rename this : `Total number of elements available to the prover.` The term `approximate size` may confuse the reader.

Reply:
I have an issue with "total" (as `|Sp| >= np` only), but if you are mitigating it with "available". I'll update it now.